apps/docs/pages/guides/platform/custom-postgres-config.mdx

```diff
@@ -0,0 +1,75 @@
+
+<Admonition type="note">
+
+Custom Postgres Config allows for advanced control over your database. Using it to set values that are inappropriate for your workload or compute add-on could cause severe performance degredation, or project instability.
+
+</Admonition>
+
+## Custom Postgres Config
+
+While most Postgres parameters can be configured from [within SQL](https://www.postgresql.org/docs/current/config-setting.html#CONFIG-SETTING-SQL-COMMAND-INTERACTION), some parameters must either be set using a config file, or require superuser access. Custom Postgres Config allows you to configure such parameters.
+
+From the perspective of Postgres, config overrides specified using this feature appear to be included in the global configuration file. Role or database specific configuration could override them for some scenarios; please refer to the [Postgres docs](https://www.postgresql.org/docs/current/) for each parameter for additional details.
+
+### Supported Parameters
+
+The following parameters are available for overrides:
+
+1. `effective_cache_size`
+1. `maintenance_work_mem`
+1. `max_connections`
+1. `max_parallel_maintenance_workers`
+1. `max_parallel_workers_per_gather`
+1. `max_parallel_workers`
+1. `max_worker_processes`
+1. `session_replication_role`
+1. `shared_buffers`
+1. `statement_timeout`
+1. `work_mem`
+
+### Setting config using the CLI
+
+To get started:
+
+1. [Install](/docs/guides/resources/supabase-cli) the Supabase CLI 1.69.0+.
+1. [Log in](/docs/guides/resources/supabase-getting-started/local-development#log-in-to-the-supabase-cli) to your Supabase account using the CLI.
+
+The `postgres config` command of the CLI can be used for setting configuration parameters:
+
+```bash
+$ supabase --experimental --project-ref <project-ref> postgres config update --config max_parallel_workers:6 --config shared_buffers:250MB
+- Custom Postgres Config -
+Config                  |Value    |
+shared_buffers          |250MB    |
+max_parallel_workers    |6        |
+- End of Custom Postgres Config -
+```
+
+By default, the CLI will merge any provided config overrides with any existing ones. The `--replace-existing-overrides` flag can be used to instead force all existing overrides to be replaced with the ones being provided:
+
+```bash
+$ supabase --experimental --project-ref <project-ref> postgres config update --config max_parallel_workers:3 --replace-existing-overrides
+
+- Custom Postgres Config -
+Config                  |Value    |
+max_parallel_workers    |3        |
+- End of Custom Postgres Config -
+```
+
+## Considerations
+
+1. The Postgres cluster will be restarted in order to change the configuration being used. This will cause momentary disruption of connections to the database; in most cases this should not take more than a few seconds.
```

Comment:
```suggestion
1. The Postgres cluster will be restarted in order to change the configuration being used. This will cause momentary disruption to existing database connections; in most cases this should not take more than a few seconds.
```